src/UI/Internal/Utils/Element.elm

```diff
@@ -16,13 +16,13 @@ tuplesToStyles ( k, v ) =
     Element.htmlAttribute <| HtmlAttrs.style k v
 
 
-ellipsisAttrs : Int -> String -> List (Attribute msg)
-ellipsisAttrs lineHeightSize titleContent =
-    [ ( "overflow", "visible" )
-    , ( "display", "block" )
-    , ( "white-space", "normal" )
-    , ( "line-height", String.fromInt lineHeightSize ++ "px" )
-    ]
+overflowAttrs : Int -> List ( String, String ) -> String -> List (Attribute msg)
+overflowAttrs lineHeightSize attrs titleContent =
+    attrs
+        |> (++)
```

Comment:
:thinking: `(++)` is an O(n) operation, It's better to do it in:
```
|> (::) ("display", "block") 
|> (::) ("line-height", String.fromInt lineHeightSize ++ "px") 
```
These are O(1) operations (and AOT reductive, not sure if Elm will do it tought)!